src/ra_lib.erl

```diff
@@ -337,6 +347,72 @@ lists_take(_N, [], Acc) ->
 lists_take(N, [H | T], Acc) ->
     lists_take(N-1, T, [H | Acc]).
 
+
+is_dir(Dir) ->
+    case prim_file:read_file_info(Dir) of
+        {ok, #file_info{type=directory}} ->
+            true;
+        _ ->
+            false
+    end.
+
+is_file(File) ->
+    case prim_file:read_file_info(File) of
+        {ok, #file_info{type = directory}} ->
```

Comment:
Just a note: this behavior is consistent with that of `filelib`: https://www.erlang.org/doc/man/filelib.html#is_file-1